pandas/io/formats/format.py

```diff
@@ -217,7 +217,7 @@ def to_string(self) -> str:
 
         fmt_values = self._get_formatted_values()
 
-        fmt_values = ["{i}".format(i=i) for i in fmt_values]
+        fmt_values = [f"{i}" for i in fmt_values]
```

Comment:
```suggestion
        fmt_values = [str(i) for i in fmt_values]
```

although should be able to remove line entirely since _get_formatted_values on line above returns List[str]

Reply:
yep, let's see if we can remove this entirely